frontend/src/features/documents/documentUpload/DocumentUploadContainer.tsx

```diff
@@ -35,8 +40,16 @@ export const DocumentUploadContainer: React.FunctionComponent<IDocumentUploadCon
     fetch();
   }, [getDocumentTypes, isMounted]);
 
-  const onDocumentTypeChange = (param: any) => {
-    // Todo: This will guide the retrieval of the metadata types
+  const onDocumentTypeChange = async (param: any) => {
```

Comment:
please set the param type to something other than any.

Reply:
done